CONTRIBUTING.md

```diff
@@ -192,6 +192,9 @@ This has the following keys:
 * `apt_dependencies` (default: `[]`): A list of Ubuntu APT packages
   that need to be installed for stubtest to run successfully. These are
   usually packages needed to pip install the implementation distribution.
+* `platforms` (default: `["linux"]`): A list of supported OSes.
+  Can contain `win32`, `linux`, and `darwin` values.
+  If not specified, only `linux` is considered supported.
```

Comment:
We should specify that this should be set only if there's some specific reason to test on multiple OSes, because we don't want useless duplicate stubtest runs for all pure-Python packages.